responses/matchers.py

```diff
@@ -121,22 +122,30 @@ def json_params_matcher(
     def match(request: PreparedRequest) -> Tuple[bool, str]:
         reason = ""
         request_body = request.body
-        params_dict = params or {}
+        json_params = (params or {}) if not isinstance(params, list) else (params or [])
```

Comment:
```suggestion
        json_params = (params or {}) if not isinstance(params, list) else params
```

Do we need the second `or`? If params is a list that is falsey, it is already an empty list.

Reply:
you are right, thanks